.github/workflows/pre-submit.e2e.generic.default.yml

```
@@ -39,3 +39,33 @@ jobs:
           BINARY: "binary-name"
           PROVENANCE: ${{ needs.build.outputs.provenance-name }}
         run: ./.github/workflows/scripts/pre-submit.e2e.generic.default.sh
+
+  build-continue-error:
+    permissions:
+      id-token: write # For signing.
+      contents: write # For asset uploads.
+      actions: read # For reading workflow info.
+    uses: ./.github/workflows/generator_generic_slsa3.yml
+    with:
+      # echo "2e0390eb024a52963db7b95e84a9c2b12c004054a7bad9a97ec0c7c89d4681d2    binary-name" | base64 -w0
+      base64-subjects: "MmUwMzkwZWIwMjRhNTI5NjNkYjdiOTVlODRhOWMyYjEyYzAwNDA1NGE3YmFkOWE5N2VjMGM3Yzg5ZDQ2ODFkMiAgICBiaW5hcnktbmFtZQo="
+      compile-generator: true
+      continue-on-error: true
+
+  verify-continue-error:
+    # NOTE: this name is used as the status check name and by protected
+    # branches for required status checks. It should have a unique name among
+    # other pre-submits.
+    name: verify generic provenance
```

This job name needs to be unique among all pre-submit jobs. Give this a different name so we can add it to protected branch checks.
```suggestion
    name: verify generic provenance (continue-on-error)
```